hieradata/site/ls.yaml

```diff
@@ -1,110 +1,143 @@
 ---
-# IT Services Layout
-# Network List at La Serena Base camp:
-#       * 139.229.136.0/24 - Users' Network
-#
-# Services List
-# DHCP Master Server            : 139.229.136.23
-# DNS Master                    : 139.229.136.23
-# Graylog Master                : 139.229.136.36
-# Grafana                       : 139.229.136.37
-# InfluxDB Master               : 139.229.136.38
-# Puppet Master                 : 139.229.136.22 (Not deployed yet)
-rsyslog::client::global_config:
+lookup_options:
+  accounts::user_list:
+    merge:
+      strategy: "deep"
+
+easy_ipa::ipa_master_fqdn: "ipa1.ls.lsst.org"
+
+# Rsyslog configuration is based on the default rsyslog.conf shipping with CentOS 7.7
+rsyslog::config::global_config:
```

Comment:
I don't want to make it difficult to get this PR merged but is there any reason we can't set the rsyslog configuration at the org level?  There is nothing working in Tucson now.

Reply:
Implemented.